applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_interface_element.cpp

```diff
@@ -1291,4 +1306,95 @@ KRATOS_TEST_CASE_IN_SUITE(InterfaceElement_CheckDoesNotThrowWhenElementIsNotActi
+    r_model_part.AddNodalSolutionStepVariable(CAUCHY_STRESS_VECTOR);
+    PointerVector<Node> nodes;
+    nodes.push_back(r_model_part.CreateNewNode(1, 0.0, 0.0, 0.0));
+    nodes.push_back(r_model_part.CreateNewNode(2, 0.0, -1.0, 0.0));
+    nodes.push_back(r_model_part.CreateNewNode(3, 1.0, 0.0, 0.0));
+    // properties for neighbour elements
+    const auto p_properties = std::make_shared<Properties>();
+    p_properties->SetValue(CONSTITUTIVE_LAW, std::make_shared<GeoIncrementalLinearElasticLaw>(
+                                                 std::make_unique<PlaneStrain>()));
+    p_properties->SetValue(YOUNG_MODULUS, 1.000000e+07);
+    p_properties->SetValue(POISSON_RATIO, 0.000000e+00);
+    // create a triangle neighbour element
+    auto p_neighbour_element = ElementSetupUtilities::Create2D3NElement(2, nodes, p_properties);
+    ProcessInfo dummy_process_info;
+    p_neighbour_element->Initialize(dummy_process_info);
+    std::vector<ConstitutiveLaw::StressVectorType> r_stress_vectors;
+    ConstitutiveLaw::StressVectorType              stress_vector(4);
+    stress_vector <<= 3.0, 13.0 / 6.0, 4.0, 1.0;
+    r_stress_vectors.emplace_back(stress_vector);
+    r_stress_vectors.emplace_back(stress_vector);
+    stress_vector <<= 3.0, 8.0 / 3.0, 4.0, 1.0;
+    r_stress_vectors.emplace_back(stress_vector);
+
+    p_neighbour_element->SetValuesOnIntegrationPoints(CAUCHY_STRESS_VECTOR, r_stress_vectors, dummy_process_info);
+
+    nodes.clear();
+    nodes.push_back(r_model_part.pGetNode(1));
+    nodes.push_back(r_model_part.pGetNode(3));
+    nodes.push_back(r_model_part.CreateNewNode(4, 0.0, 1.0, 0.0));
+    nodes.push_back(r_model_part.CreateNewNode(5, 1.0, 1.0, 0.0));
+    const auto     p_geometry       = std::make_shared<LineInterfaceGeometry2D2Plus2Noded>(nodes);
+    constexpr auto normal_stiffness = 20.0;
+    constexpr auto shear_stiffness  = 10.0;
+    const auto     p_interface_properties =
+        CreateElasticMaterialProperties<InterfacePlaneStrain>(normal_stiffness, shear_stiffness);
+    auto interface_element = CreateInterfaceElementWithUDofs<Interface2D>(p_interface_properties, p_geometry);
+    Variable<GlobalPointersVector<Element>>::Type neighbours{p_neighbour_element};
+    interface_element.SetValue(NEIGHBOUR_ELEMENTS, neighbours);
+
+    // Act
+    interface_element.Initialize(dummy_process_info);
+
+    std::vector<Vector> actual_traction_vectors;
+    interface_element.CalculateOnIntegrationPoints(CAUCHY_STRESS_VECTOR, actual_traction_vectors, dummy_process_info);
+
+    // Assert
+    auto expected_traction_vector = Vector{2};
+    // answers for 1st side
+    expected_traction_vector <<= 2.0, 1.0;
+    KRATOS_EXPECT_VECTOR_RELATIVE_NEAR(actual_traction_vectors[0], expected_traction_vector, Defaults::relative_tolerance)
+    expected_traction_vector <<= 3.0, 1.0;
+    KRATOS_EXPECT_VECTOR_RELATIVE_NEAR(actual_traction_vectors[1], expected_traction_vector, Defaults::relative_tolerance)
```

Comment:
Perhaps the `UblasUtilities` class can help to write down things in a slightly more compact way:
```suggestion
    // answers for 1st side
    KRATOS_EXPECT_VECTOR_RELATIVE_NEAR(actual_traction_vectors[0], UblasUtilities::CreateVector({2.0, 1.0}), Defaults::relative_tolerance)
    KRATOS_EXPECT_VECTOR_RELATIVE_NEAR(actual_traction_vectors[1], UblasUtilities::CreateVector({3.0, 1.0}), Defaults::relative_tolerance)
```

Reply:
beautiful. done